docs/feature-gates.md

```diff
@@ -366,3 +367,8 @@ transport protocol, and port. Refer to this [document](antrea-l7-network-policy.
 
 This feature is currently only supported for Nodes running Linux, and TX checksum offloading must be disabled. Refer to
 this [document](antrea-l7-network-policy.md#prerequisites) for more information and how it can be configured.
+
+### AdminNetworkPolicy
+
+The `AdminNetworkPolicy` API (which currently includes the AdminNetworkPolicy and BaselineAdminNetworkPolicy objects)
+complements the Antrea-native policies and help cluster administrators to set security postures in a portable manner.
```

Comment:
Will we be adding more documentation as a follow-up PR (maybe in antrea-policy.md) to explain how admin policies co-exist with Antrea policies?

Reply:
Yes will open a PR for this

Reply:
Please also help to check if there is any needed to update `### NetworkPolicy commands` section in antctl.md or any antctl code changes required. I saw a description like `The source of a control plane NetworkPolicy is the original policy resource (K8s NetworkPolicy or Antrea-native Policy)`,